tox.ini

```diff
@@ -1,21 +1,23 @@
 [tox]
-envlist = {py27,py33,py34,py35,py36,pypy}-{base,optional}
+envlist = {py27,py33,py34,py35,py36,pypy}-{base,six19,optional}
 
 [testenv]
 deps =
-  flake8<3.0
-  pytest
-  pytest-expect>=1.1,<2.0
-  mock
-  base: six
-  base: webencodings
   optional: -r{toxinidir}/requirements-optional.txt
+  -r{toxinidir}/requirements-test.txt
   doc: Sphinx
```

Comment:
It looks like when this ran the tests with `six==1.9`, it'd also run the tests with the optional requirements. I'm pretty sure your changes don't do that--`six19` is a separate environment from `optional`.

I don't know offhand if that's *important*, but it seems like if we want to maintain parity, it'd be better to add a line here like:

```
six19: -r{toxinidir}/requirements-optional.txt
```

Reply:
Sorry, yes I missed that.  It would be better to achieve that by using `TOXENV=six19-optional` .